Lib/test/test_importlib/extension/test_loader.py

```diff
@@ -145,6 +145,21 @@ def test_reload(self):
             importlib.reload(module)
             self.assertIs(ex_class, module.Example)
 
+    def test_reload_faliure(self):
```

Comment:
Tests for reloading should be in [test.test_importlib.test_api](https://github.com/python/cpython/blob/master/Lib/test/test_importlib/test_api.py#L198).